internal/telemetry/tracker.go

```diff
@@ -106,3 +116,24 @@ func (t *tracker) save(event Event) error {
 	_, err = file.Write(data)
 	return err
 }
+
+func send(ctx context.Context, events *[]Event) error {
+	if config.Service() != config.CloudService {
+		// Only send events to Atlas - not to AtlasGov or OpsManager or CloudManager
+		return nil
+	}
+	s, err := store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx), store.Telemetry())
+	if err != nil {
+		return err
+	}
+	client, err := s.GetAtlasClient()
+	if err != nil {
+		return err
+	}
+	request, err := client.NewRequest(ctx, http.MethodPost, urlPath, &events)
```

Comment:
Let's move this method to be inside the store like other endpoints, this would give us the advantage of being able to mock it and unit test this function properly

Reply:
OK, I have moved the "send" function to the store package (and renamed it SendEvents).